dataset/dictionary/locations.json5

```diff
@@ -1112,90 +1122,105 @@
   {
     en: "Vimara Village",
     ja: "ヴィマラ村",
+    zhCN: "维摩庄",
     pronunciationJa: "ヴィマラむら",
     tags: [ "sumeru", "location" ],
   },
   {
     en: "Devantaka Mountain",
     ja: "デーヴァーンタカ山",
+    zhCN: "降诸魔山",
     pronunciationJa: "デーヴァーンタカやま",
     tags: [ "sumeru", "location" ],
   },
 
   {
     en: "Lokapala Jungle",
     ja: "ローカパーラ密林",
+    zhCN: "护世森",
     pronunciationJa: "ローカパーラみつりん",
     tags: [ "sumeru", "location" ],
   },
   {
     en: "Mawtiyima Forest",
     ja: "マウティーマ稠林",
+    zhCN: "无郁稠林",
     pronunciationJa: "マウティーマちゅうりん",
     tags: [ "sumeru", "location" ],
   },
   {
     en: "The Palace of Alcazarzaray",
     ja: "アルカサルザライパレス",
+    zhCN: "",
     tags: [ "sumeru", "location", "facility" ],
   },
   {
     en: "Chatrakam Cave",
     ja: "チャトラカム洞窟",
+    zhCN: "卡萨扎莱宫",
```

Comment:
Maybe this is the translation for The Palace of Alcazarzaray.
![](https://user-images.githubusercontent.com/107226297/187026900-3d1da85d-e9e7-421b-95b3-bd57d319d1df.png)
![](https://user-images.githubusercontent.com/107226297/187026901-c11e8107-0c39-4a35-92f3-26d76ff23fd4.png)

Reply:
I'm sorry, it's misplaced here.